src/transforms.jl

```diff
@@ -216,6 +216,7 @@ include("transforms/filter.jl")
 include("transforms/rename.jl")
 include("transforms/identity.jl")
 include("transforms/center.jl")
+include("transforms/coalesce.jl")
```

Comment:
Same here, before identity.jl

Reply:
Made in [76d50d3](https://github.com/JuliaML/TableTransforms.jl/pull/40/commits/76d50d3a5bf0473adec685a23b83c0424881908c)